packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -162,6 +72,7 @@ export default function ListBoxWithTags(props) {
             isDisabled,
```

Comment:
replace with `<ListBoxTrigger />` (see the comment above)

Reply:
I was going to call it `<TriggerWithTags>` 😝 but I don't think it will actually work as a component.

Reply:
I think you can do something like this:

```jsx
// the internal render of ListBox.Trigger
function Trigger(props) {
  const triggerStuff = { triggerA: 1, triggerB: 2 };
  return props.children(triggerStuff);
}

function CustomTrigger(props) {
  return `${props.triggerA} ${props.triggerB} ${props.x} ${props.y}`;
}

export default function App() {
  const customApp = (extraProps) => (props) => {
    return <CustomTrigger {...props} {...extraProps} />;
  };
  
  return (
    <div className="App">
      <Trigger>{customApp({ x: "x", y: "y" })}</Trigger>
    </div>
  );
}
```

https://codesandbox.io/s/nervous-lamarr-kc25b?file=/src/App.js:24-504

Reply:
Cool, this is very similar to the approach Andrei suggested offline in a little chat we had 👍 👍